src/components/wallet-select/wallets.ts

```diff
@@ -16,35 +22,28 @@ const wallets: { [walletType: string]: Wallet } = {
   [WalletType.Metamask]: {
     type: WalletType.Metamask,
     name: 'Metamask',
-    imageSource: 'metamask.svg',
+    imageSource: metamaskIcon,
   },
   [WalletType.WalletConnect]: {
     type: WalletType.WalletConnect,
     name: 'Wallet Connect',
-    imageSource: 'walletconnect.svg',
+    imageSource: walletConnectIcon,
   },
   [WalletType.Coinbase]: {
     type: WalletType.Coinbase,
     name: 'Coinbase Wallet',
-    imageSource: 'coinbasewallet.svg',
+    imageSource: coinbaseIcon,
   },
   [WalletType.Fortmatic]: {
     type: WalletType.Fortmatic,
     name: 'Fortmatic',
-    imageSource: 'fortmatic.svg',
+    imageSource: fortmaticIcon,
   },
   [WalletType.Portis]: {
     type: WalletType.Portis,
     name: 'Portis',
-    imageSource: 'portis.svg',
+    imageSource: portisIcon,
   },
 };
 
-for (const wallet in wallets) {
-  wallets[wallet].imageSource = [
-    'https://res.cloudinary.com/fact0ry-dev/image/upload/v1637005920/zero-assets/wallet-providers',
```

Comment:
yes, i think we can just update the url here with the new one (provided we have uploaded the icons there).

Reply:
Yup we need to upload the icons. Requested this in the Jira task earlier this week, just waiting for Joel/Dale to have some free time to do so, think other work is higher priority at the mo, but keeping tabs on this.